### ISSUE TITLE
wrong assertion in db_bench_toku ReadSequential

### ISSUE THREAD

**Issue by enigma-yang** [hyc/leveldb#5]
``` c
  void ReadSequential(ThreadState *thread) {
    // ignore
    assert(r == 0);
    while (r != DB_NOTFOUND) {
      r = cursor->c_getf_next(cursor, 0, &CursorAccumulate, thread);
    }
    assert(r == 0);
    cursor->c_close(cursor);
    txn->abort(txn);
  }
```

After while loop, r is DB_NOTFOUND which is not zero. So the second assertion is invalid. In debug mode, this also causes benchmark crash without closing cursor, which further crash the whole database.
Same thing happens in ReadReverse.

**Comment by hyc** [comment]
Thanks. In practice, the benchmarks are compiled with -DNDEBUG so all of the asserts are nops, so this doesn't affect timing runs.

**Comment by hyc** [comment]
Fixed by 0a6455d91c72c3e139bd876ea20753deb5c3383d
